src/components/Button/Button.module.css

```diff
@@ -168,12 +169,12 @@
 }
 
 .secondary.includeHoverAnimations:hover .textLabel {
-  color: var(--white);
+  color: var(--darkmode-static-white);
```

Comment:
I don't see `--darkmode-static-white` in colors.css, am I missing it or does it only exist in audius-client?

Also could you explain the difference between `--darkmode-static-white` and `--static-white` to me? I am confusion